GateMaterials.db

```diff
@@ -476,16 +664,241 @@ Testis: d=1.04 g/cm3 ; n=9
+        +el: name=Oxygen     ; f=0.1733
+        +el: name=Magnesium  ; f=0.1136
+        +el: name=Silicon    ; f=0.0072
+        +el: name=Chlorine   ; f=0.0015
+
+CdTe:   d=5.85 g/cm3 ; n=2; state=solid
+        +el: name=Cadmium    ; f=0.468358
+        +el: name=Tellurium  ; f=0.531642
+
+PMMA:   d=1.195 g/cm3; n=3 ; state=solid
         +el: name=Hydrogen   ; f=0.080541
         +el: name=Carbon     ; f=0.599846
         +el: name=Oxygen     ; f=0.319613
 
-Epoxy:  d=1.0 g/cm3; n=3; state=Solid
+PMMA2:   d=1.19 g/cm3 ; n=3 ; state=solid
+        +el: name=Hydrogen   ; f=0.080538
+        +el: name=Carbon     ; f=0.599848
+        +el: name=Oxygen     ; f=0.319614
+
+PMMA3:   d=1.195 g/cm3; n=3 ; state=solid
+        +el: name=Hydrogen   ; f=0.080541
+        +el: name=Carbon     ; f=0.599846
+        +el: name=Sulfur     ; f=0.319613
+
+Epoxy:  d=1.0 g/cm3; n=3; state=solid
         +el: name=Carbon     ; n=1
         +el: name=Hydrogen   ; n=1
         +el: name=Oxygen     ; n=1
 
-Carbide: d=15.8 g/cm3; n=2 ; state=Solid
+Carbide: d=15.8 g/cm3; n=2 ; state=solid
         +el: name=Tungsten   ; n=1
         +el: name=Carbon     ; n=1
+
+ABS: d=1.10 g/cm3; n=3 ; state=solid
+        +el: name=Hydrogen   ; n=17
+        +el: name=Carbon     ; n=15
+        +el: name=Nitrogen   ; n=1
+
+Tumor: d=1 g/cm3 ; n=11
+        +el: name=Hydrogen   ; f=0.120
+        +el: name=Carbon     ; f=0.640
+        +el: name=Nitrogen   ; f=0.008
+        +el: name=Oxygen     ; f=0.229
+        +el: name=Phosphor   ; f=0.002
+        +el: name=Calcium    ; f=0.001
+        +el: name=Scandium   ; f=0.0
+        +el: name=Titanium   ; f=0.0
+        +el: name=Vandium    ; f=0.0
+        +el: name=Chromium   ; f=0.0
+        +el: name=Manganese  ; f=0.0
+
+FR4_epoxy:  d=1.85 g/cm3; n=4; state=solid
+        +el: name=Carbon     ; f=0.6419
+        +el: name=Hydrogen   ; f=0.0643
+        +el: name=Oxygen     ; f=0.2036
+        +el: name=Chlorine   ; f=0.0902
+
+FR4:  d=1.85 g/cm3; n=11; state=solid
+        +el: name=Silicon    ; f=0.15
+        +el: name=Oxygen     ; f=0.36
+        +el: name=Calcium    ; f=0.08
+        +el: name=Aluminium  ; f=0.04
+        +el: name=Magnesium  ; f=0.01
+        +el: name=Boron      ; f=0.01
+        +el: name=Potassium  ; f=0.01
+        +el: name=Carbon     ; f=0.27
+        +el: name=Hydrogen   ; f=0.03
+        +el: name=Nitrogen   ; f=0.03
+        +el: name=Sodium     ; f=0.01
+
+FR4_Copper: d=2.2998 g/cm3; n=4; state=solid
+        +el: name=Gold       ; f=0.01
+        +el: name=Nickel     ; f=0.02
+        +el: name=Copper     ; f=0.21
+        +mat: name=FR4       ; f=0.76
+
+FR4_Copper_epoxy: d=2.2998 g/cm3; n=7; state=solid
+        +el: name=Gold       ; f=0.0131
+        +el: name=Nickel     ; f=0.0182
+        +el: name=Copper     ; f=0.2134
+        +el: name=Carbon     ; f=0.4848
+        +el: name=Hydrogen   ; f=0.0486
+        +el: name=Oxygen     ; f=0.1538
+        +el: name=Chlorine   ; f=0.0681
```

Comment:
Any clue what's FR4?

Reply:
https://en.wikipedia.org/wiki/FR-4

looks like a material typically used for circuit boards. maybe important for simulation of SiPMs?

i think i will keep these materials for now